subprojects/common/math/src/main/kotlin/com/avito/math/Percent.kt

```diff
@@ -23,32 +25,48 @@ public fun Double.fromZeroToOnePercent(): Percent {
 @Suppress("MagicNumber")
 public fun Float.fromZeroToOnePercent(): Percent {
     require(this in 0.0..1.0) { "Trying to convert $this to percents; should be in [0.0..1.0] range" }
-    return NumberPercent(this * 100)
+    return BigDecimalTwoDigitsPrecisionPercent.create(this * 100)
 }
 
 public fun Int.fromZeroToHundredPercent(): Percent {
     require(this in 0..100) { "Trying to convert $this to percents; should be in [0..100] range" }
-    return NumberPercent(this)
+    return BigDecimalTwoDigitsPrecisionPercent.create(this)
 }
 
 public interface Percent {
 
-    public fun twoDigitsString(): String
+    public fun roundToTwoDigitsPrecision(): String
```

Comment:
Methods `roundTo...` are not simmetrical in return types:

rountToInt() --> Int
roundToTwoDigitsPrecision() --> String

Maybe this should be `format...`

Reply:
Agree